lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -604,9 +605,22 @@ func asRustBackedSlice(data *C.uint8_t, length int) []byte {
 	return unsafe.Slice(uptr, length)
 }
 
+func value(handle C.uintptr_t) (value any, err error) {
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
What types of panic are we trying to recover from here?

Reply:
`handle.Value` will panic if it was already `Delete()`d. There's probability that our interaction between multiple goroutines and tokio runtime on rust side will lead to that situation - I hit it once, when I wasn't able to connect to the instance due to firewall and then reloaded page and tried to log in again. I wasn't able to reproduce it since. Just to be sure we won't kill the whole instance it's better to do the check